ocaml/parsing/parser.mly

```diff
@@ -3529,10 +3529,12 @@ pattern_no_exn:
       { let loc = $loc(label) in
         Some label, mkpatvar ~loc label }
   | TILDE LPAREN label = LIDENT COLON cty = core_type RPAREN %prec COMMA
-      { let loc = $loc(label) in
-        let pat = mkpatvar ~loc label in
-        Some label, mkpat_opt_constraint ~loc pat (Some cty) }
+      { let lbl_loc = $loc(label) in
+        let pat_loc = $startpos($2), $endpos in
+        let pat = mkpatvar ~loc:lbl_loc label in
+        Some label, mkpat_opt_constraint ~loc:pat_loc pat (Some cty) }
 
+(* If changing this, don't forget to change its copy just above. *)
```

Comment:
Should there be a converse comment just above, too?

Reply:
The other instance is already preceded by a comment describing the existence of a copy, so I think it's already covered.